### ISSUE TITLE
Frozen object name collision

### ISSUE THREAD

**Issue by ondras** [rollup/rollup#943]
```js
// main.js
import * as log from "./log.js";
window.xxx = log;

log.log2();
```

```js
// log.js
export default function log(x) {
   console.log(x);
}

export function log2() {
  log(2);
}
```

Rolls up to:

```js
'use strict';

function log(x) {
   console.log(x);
}

function log2() {
  log(2);
}


var log = Object.freeze({
	default: log,
	log2: log2
});

window.xxx = log;

log2();
```

  * note that the frozen `log` object conflicts with the (internal) `log` function
  * try to execute the code; `log2` fails

**Comment by Rich-Harris** [comment]
This was fixed by #934, and is [working with 0.35.11](http://rollupjs.org/?shareable=JTdCJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmNqcyUyMiUyQyUyMm1vZHVsZU5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmdsb2JhbHMlMjIlM0ElN0IlN0QlN0QlMkMlMjJtb2R1bGVzJTIyJTNBJTVCJTdCJTIybmFtZSUyMiUzQSUyMm1haW4uanMlMjIlMkMlMjJjb2RlJTIyJTNBJTIyaW1wb3J0JTIwKiUyMGFzJTIwbG9nJTIwZnJvbSUyMCU1QyUyMi4lMkZsb2cuanMlNUMlMjIlM0IlNUNud2luZG93Lnh4eCUyMCUzRCUyMGxvZyUzQiU1Q24lNUNubG9nLmxvZzIoKSUzQiUyMiU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJsb2cuanMlMjIlMkMlMjJjb2RlJTIyJTNBJTIyZXhwb3J0JTIwZGVmYXVsdCUyMGZ1bmN0aW9uJTIwbG9nKHgpJTIwJTdCJTVDbiUyMCUyMCUyMGNvbnNvbGUubG9nKHgpJTNCJTVDbiU3RCU1Q24lNUNuZXhwb3J0JTIwZnVuY3Rpb24lMjBsb2cyKCklMjAlN0IlNUNuJTIwJTIwbG9nKDIpJTNCJTVDbiU3RCUyMiU3RCU1RCU3RA==), so I'll close this 🎉

**Comment by ondras** [comment]
Ah, my bad, sorry for not checking this more thoroughly.
